pom.xml

```diff
@@ -602,6 +657,107 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>push-docker-image</id>
+            <build>
+                <pluginManagement>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>0.25.0</version>
+
+                        <configuration>
+                            <registry>registry.hub.docker.com/ws1connectors</registry>
+                            <authConfig>
+                                <username>appplatform</username>
+                                <password>dockerhub_8976</password>
+                            </authConfig>
+
+                            <images>
+                                <image>
+                                    <name>${project.artifactId}</name>
+                                    <alias>${project.artifactId}</alias>
+                                    <build>
+                                        <assembly>
+                                            <inline>
+                                                <dependencySets>
+                                                    <dependencySet>
+                                                        <useProjectArtifact>true</useProjectArtifact>
+                                                        <includes>
+                                                            <include>com.vmware.card-connectors:${project.artifactId}</include>
+                                                        </includes>
+                                                    </dependencySet>
+                                                </dependencySets>
+                                            </inline>
+                                            <targetDir>/opt/vmware/${connector.name}</targetDir>
+                                        </assembly>
+                                        <from>openjdk:8u151-jre</from>
```

Comment:
You shouldn't need to put all this at the root. `connectors/pom.xml` should work.

Reply:
Moved the contents to ```connectors``` ```pom.xml```.